code/processes/wdb.q

```diff
@@ -477,7 +479,10 @@ replayupd:{[f;t;d]
 	/- if the data count is greater than the threshold, then flush data to disk
 	if[(rpc:count[value t]) > lmt:maxrows[t];
 		.lg.o[`replayupd;"row limit (",string[lmt],") exceeded for ",string[t],". Table count is : ",string[rpc],". Flushing table to disk..."];
-		savetables[savedir;getpartition[];0b;t]]	
+		/- if datastriping is on then filter before savedown to the tailDB, if not save down to wdbhdb
+    $[.ds.datastripe;
+     (.ds.savetablesoverperiod[.ds.td;;.z.p+10:00;.z.p]each .wdb.tablelist[];.ds.applyfilters[ enlist t;.sub.filterdict]);
```

Comment:
might want to use `[]` here to wrap statements together as a code block

```suggestion
     [.ds.savetablesoverperiod[.ds.td;;.z.p+10:00;.z.p]each .wdb.tablelist[];.ds.applyfilters[ enlist t;.sub.filterdict]];
```

with (), statements will be executed right-to-left, which may be unintended/unclear

Reply:
I want the second statement to run before the first, but will change it to [] and swap it to make it clearer